.github/workflows/format_check.yml

```
@@ -87,3 +90,35 @@ jobs:
             {\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"<@${{ steps.match-github-to-slack-user.outputs.slack_user_ids }}> \n\"}},
             {\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\" :octavia-shocked: <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}|View Action Run> :octavia-shocked: \n\"}},
             {\"type\":\"divider\"}]}
+
+  format-check-from-forks:
+    # Same-named job as above, in order to ensure 'required checks' pass either way.
+    # This should run all the same checks as above, except not requiring any credentials.
+    # This may be slower than the version that runs with creds, due to the affect of
+    # caching.
+    name: "Check for formatting errors"
+    if: >
+      github.event.pull_request.head.repo.fork == true
+    runs-on: ubuntu-latest
+    steps:
+      # We have no creds. Ignore docker caching and just run the CLI.
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 1
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Install `airbyte-ci` CLI
+        # We use the stable `airbyte-ci` from `master`. Ignores any CI changes
+        # local to this repo.
+        run: >
+          pip install
```

```suggestion
     - name: Install Airbyte CI from binary 
       id: install-airbyte-ci-binary 
       shell: bash 
       run: | 
         curl -sSL "https://connectors.airbyte.com/airbyte-ci/releases/ubuntu/latest/airbyte-ci" --output airbyte-ci-bin 
         sudo mv airbyte-ci-bin /usr/local/bin/airbyte-ci 
         sudo chmod +x /usr/local/bin/airbyte-ci
```

Binary install from https://github.com/airbytehq/airbyte/blob/dfe0b74a4d6f5cfc15a910cfb4c89821f7d2394c/.github/actions/install-airbyte-ci/action.yml#L35-L42